src/custom/state/claim/hooks/index.ts

```diff
@@ -161,16 +174,19 @@ export function useUserHasAvailableClaim(account: Account): boolean {
   return availableClaims.length > 0
 }
 
-export function useUserUnclaimedAmount(account: string | null | undefined): CurrencyAmount<Token> | undefined {
+export function useUserUnclaimedCurrentAmount(account: string | null | undefined): CurrencyAmount<Token> | undefined {
```

Comment:
Why the renaming? We cannot use the original `useUserUnclaimedAmount` instead as it'll not give us meaningful data. Might as well keep the same name.

Reply:
this was my comment, i wanted separate hooks:
1. `useUserUnclaimedAmount` (no pending amounts subtrated)
2. `useUserUnclaimedCurrentAmount` with real unclaimed amount - pending amounts (as above)

Reply:
where `2` just uses output from `1` and returns amount